pkg/test/main_entry.go

```diff
@@ -25,8 +25,22 @@ import (
 func MainEntry(m *testing.M) {
 	fopts := &frameworkOpts{}
 	fopts.addToFlagSet(flag.CommandLine)
+	// controller-runtime registers the --kubeconfig flag in client config
+	// package:
+	// https://github.com/kubernetes-sigs/controller-runtime/blob/v0.5.2/pkg/client/config/config.go#L39
+	//
+	// If this flag is not registered, do so. Otherwise retrieve its value.
+	kcFlag := flag.Lookup(KubeConfigFlag)
+	if kcFlag == nil {
+		flag.StringVar(&fopts.kubeconfigPath, KubeConfigFlag, "", "path to kubeconfig")
+	}
+
 	flag.Parse()
 
+	if kcFlag != nil {
+		fopts.kubeconfigPath = kcFlag.Value.String()
+	}
+
 	f, err := newFramework(fopts)
 	if err != nil {
 		log.Fatalf("Failed to create framework: %v", err)
```

Comment:
Also, we are testing it just [here](https://github.com/operator-framework/operator-sdk/blob/321ae57a3650e1b406d938370ffbafbdc0ebf6a0/hack/tests/subcommand.sh#L18). WDYT about add a go test with the user scenario?

> And in my test case, I am using a library which is also using controller-runtime/operator-sdk
> github.com/RHsyseng/operator-utils/pkg/utils/openshift
> 
>  Method I am calling: openshift.IsOpenShift(framework.KubeConfig)

Reply:
I concur, we should probably add a test.